src/routes/github/github-router.ts

```diff
@@ -41,7 +51,8 @@ subRouter.post("/webhooks",
 	WebhookReceiverPost);
 
 // Create-branch is seperated above since it currently relies on query param to extract the jirahost
-subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, GithubCreateBranchRouter);
+// Todo able to move under the jirasymmetric middleware once flag completed
+subRouter.use("/create-branch", JiraHostFromQueryParamMiddleware, GithubServerAppMiddleware, GithubAuthMiddleware, csrfMiddleware, maybeJiraSymmetricJwtMiddleware, GithubCreateBranchRouter);
 
```

Comment:
`maybeJiraSymmetricJwtMiddleware` should be not put after `JiraHostFromQueryParamMiddleware`? All other middlewares such as `GithubServerAppMiddleware` and `GithubAuthMiddleware` require `jiraHost` in res.locals